application/pom.xml

```diff
@@ -26,4 +26,9 @@
         </dependency>
     </dependencies>
 
+    <properties>
+        <maven.compiler.source>6</maven.compiler.source>
+        <maven.compiler.target>1.6</maven.compiler.target>
+    </properties>
```

Comment:
Can it be compiled with a more recent version of Java? If it is, do it in a new PR, because is a relevant change.